homeassistant/components/switchbee/config_flow.py

```diff
@@ -45,6 +45,7 @@ async def validate_input(hass: HomeAssistant, data: dict[str, Any]):
 
         raise CannotConnect from exp
 
+    assert isinstance(api.mac, str)
```

Comment:
It is more efficient to check for `None`:
```suggestion
    assert api.mac is not None
```